inception-project/inception
# #4641 - Ability to assign names to endpoints of relations

**What's in the PR**
- Added a new relation endpoint feature support
- Allow editing relation endpoint names (only UI names)
- Create these features when creating new relation layers (existing layers don't get them)

**How to test manually**
* Try working with existing relation layers
* Try creating new projects with new relation layers
* Try changing the names of the relation endpoints and check that the change reflects e.g. in the annotation detail panel

**Automatic testing**

**Documentation**
